### DISCUSSION TITLE
problems installing youtube -dl through Homebrew

### DISCUSSION THREAD

**Discussion by parapluie7** [Homebrew/discussions#4058] [Getting started]
I had a problem installing youtube-dl through Homebrew on macOS Monterey. I installed Homebrew, mpv, and youtube -d but after I installed youtube-dl I had this message. HOMEBREW_NO_INSTALL_CLEANUP. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`). Have any of you had the same issue?  Please I need your help, thank you in advance :-)

**Comment by p-linnane** [comment]
This is just giving you hints on how you can change Homebrew's behavior if you'd like. It shouldn't have any effect on installing formulae or casks.

From the [man page](https://docs.brew.sh/Manpage#environment):
>- HOMEBREW_NO_ENV_HINTS 
>If set, do not print any hints about changing Homebrew’s behaviour with environment variables.

>- HOMEBREW_NO_INSTALL_CLEANUP 
If set, brew install, brew upgrade and brew reinstall will never automatically cleanup installed/upgraded/reinstalled formulae or all formulae every HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS days. Alternatively, HOMEBREW_NO_CLEANUP_FORMULAE allows specifying specific formulae to not clean up.

**Reply by parapluie7** [reply]
Thank you. Do you mean that youtube-dl is installed?  And I don't have to do anything?   I need Homebrew, mpv,  youtube-dl, and also ffmpeg to make Anki flashcards with audio for learning languages.  I was afraid that I will not be able to download youtube videos in order to make the flashcards.

**Reply by p-linnane** [reply]
It should be. Run `brew list` to see what you have installed.

**Reply by parapluie7** [reply]
Formulae
ca-certificates	openssl@1.1	readline	xz
mpdecimal	python@3.11	sqlite		youtube-dl

==> Casks
mpv

**Reply by parapluie7** [reply]
Thank you so much, I was so worried.
